tpm2/src/unseal/main.c

```diff
@@ -191,6 +219,11 @@ int main(int argc, char **argv)
 
   free(default_outPath);
   kmyth_clear_and_free(outputData, outputSize);
+  if (authString != NULL)
```

Comment:
Again, if we cleared authString and ownerAuthPasswd earlier, we wouldn't have to do so here.